### PR TITLE
Update oss-license-plugin to list more libraries

### LINKED ISSUE #713: All dependent libraries are not enumerated in License Activity.
**Overview (Required)**
- The License page of this app uses [oss-licenses-plugin](https://github.com/google/play-services-plugins/tree/master/oss-licenses-plugin).
- Currently, there is a problem that all dependent libraries are not enumerated.
- The cause is probably due to the composition of the multi-module project.
- Related issues
  - google/play-services-plugins#7
  - google/play-services-plugins#25

**Steps To Reproduce**
- Run DroidKaigi 2020
- Open `License` page from `About` page.
- Only some dependent libraries are listed

**Env(specified version):**
- `play-services-oss-licenses` -> `17.0.0`
- `oss-licenses-plugin` -> `0.10.0`

**Additional context**
- Previous isssue and pull request #555 / #687

### PR DESCRIPTION
**Pull request by koji-1009**
## Issue
- close #713

## Overview (Required)
- Update OSS licenses Gradle plugin 
  - More libraries are listed than before!

## Links
- <https://developers.google.com/android/guides/releases#march_09_2020>

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
LGTM！！📝

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
     steps:
       - checkout
       - restore_and_save_gradle_cache
-      - run: ./gradlew android-base:assembleDebug --offline
+      - run: ./gradlew android-base:assembleDebug
```
Comment (koji-1009) [inline .circleci/config.yml]: Same problem as #715 😢 

OSS plugin gets "all" library dependencies.  [code](https://github.com/google/play-services-plugins/pull/128/files#diff-0371e054166e1719100f44f7aaf94742R220)
So,  crash build like [1951 build](https://app.circleci.com/pipelines/github/DroidKaigi/conference-app-2020/1207/workflows/c51ef3a2-b095-491f-8958-1559f07cfefe/jobs/1951).

**Comment by takahirom** [comment]
🙆

**Comment by koji-1009** [comment]
Thanks!

**Pull request merged**